### DISCUSSION TITLE
OBIS ascii, how to decode multiple values within brackets

### DISCUSSION THREAD

**Discussion by jjeb-de** [arendst/Tasmota#20817] [Q&A]
Hello everybody,

I try go decode following string
19:26:58.576 : 9.4(078.2*C&073.4*C),
using Smart Meter  interface.
The descriptor
1,9.4(@1,Max. Temp. V-Lauf,°C,c_max_temp_fw,1
sets variable c_max_temp_da to 78.2,
but I do not succeed following following to get the second value 73.4
Any help appreciated

**Comment by gemu2015** [comment]
this is a very unusual format, currently not supported
normally values are separated this way which is supported.
19:26:58.576 : 9.4(078.2)(073.4C)
i will think on how to add this to the decoder.

**Reply by gemu2015** [reply]
looking at the code, i realized i had already implemented a general approach to solve this and forgot to add this to the docs.
try
1,9.4(@s&1:1,Max. Temp. V-Lauf,°C,c_max_temp_fw,1

's' = skip 
'&' = separation char
'1' = number of entries to skip

**Comment by jjeb-de** [comment]
Hi gemu2015,

works perfectly as expected, tasmota is a great framework.
Thanx for your support

Best Regards
